app/src/main/res/values/strings.xml

```diff
@@ -100,7 +100,7 @@
     <string name="settings_key_lock_barcode_orientation" translatable="false">pref_lock_barcode_orientation</string>
     <string name="settings_keep_screen_on">Keep screen on while viewing a card</string>
```

Comment:
```suggestion
    <string name="settings_keep_screen_on">Keep screen on while displaying</string>
```

Reply:
Definitely not an improvement, as the screen is only kept on while viewing a card. So now it is no longer clear what someone is changing.